### PR TITLE
Add linuxserver.io docker installation commands to documentation

### PR DESCRIPTION
**Pull request by lightlike**

### CODE REVIEW

**Comment by nickvergessen** [comment]
Hmm all of Nextclouds documentation is ubuntu/apache based. Any specific reason why we should diverge here?

**Comment by lightlike** [comment]
This took me a pretty long time to figure out (but that might be me). Wanted to help others who might have a similar problems.

If you want to keep this in line with the rest of the documentation you might want to add php7 before the command as that should also work on docker.

Maybe:
```
sudo -u www-data php7 /var/www/nextcloud/occ talk:command:add vote Poll 'php7 /var/www/nextcloud/occ talk:poll:vote {ROOM} {USER} {ARGUMENTS}' 2 3
```

**Comment by nickvergessen** [comment]
without the 7, but yeah let me change that

**Comment by nickvergessen** [comment]
Or you want to adjust your PR?

File: README.md [hunk 1]
```diff
@@ -86,8 +86,15 @@ Poll: Do you like polls?
 ## Installation
 
 1. Install the app
-2. Setup the commands (you need to use your full path to occ, also don't use `sudo -u www-data` inside the command, because it is automatically run as www-data):
+2. Setup the commands (you need to use your full path to occ, also don't use `sudo -u www-data` inside the command, because it is automatically run as www-data):  
+  
+    Normal installation:
     ```
     sudo -u www-data /var/www/nextcloud/occ talk:command:add poll Poll '/var/www/nextcloud/occ talk:poll {ROOM} {USER} {ARGUMENTS}' 2 3
     sudo -u www-data /var/www/nextcloud/occ talk:command:add vote Poll '/var/www/nextcloud/occ talk:poll:vote {ROOM} {USER} {ARGUMENTS}' 2 3
```
Comment (nickvergessen) [inline README.md:94]:
```suggestion
    sudo -u www-data php /var/www/nextcloud/occ talk:command:add poll Poll 'php /var/www/nextcloud/occ talk:poll {ROOM} {USER} {ARGUMENTS}' 2 3
    sudo -u www-data php /var/www/nextcloud/occ talk:command:add vote Poll 'php /var/www/nextcloud/occ talk:poll:vote {ROOM} {USER} {ARGUMENTS}' 2 3
```

**Comment by lightlike** [comment]
Tested on my server and those commands work nicely. (with changed paths of course)

I hopefully comitted everything ok. Never really edited a pull request.

**Pull request merged**